recipes/pymesh2/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.2.1" %}
+
+package:
+  name: pymesh2
+  version: {{ version }}
+
+source:
+  url: https://github.com/PyMesh/PyMesh/releases/download/v{{ version }}/pymesh2-{{ version }}-cp36-cp36m-linux_x86_64.whl
```

Comment:
This is not even a `manylinux1` wheel. Why not use `pip` to install this if there's an urgent need?

Reply:
Reasons... (I'll ping me in pvt and I'll explain it.)